tutorials/Numerics/TimeStepping/imex_ark.jl

```diff
@@ -0,0 +1,180 @@
+# ``
+#
+# The Butcher coefficients ``\boldsymbol{c}``, ``\boldsymbol{b}_{\nu}``, ``\boldsymbol{A}_{\nu}``, and ``\widehat{\boldsymbol{b}}_{\nu}``
+# are constrained by certain accuracy and stability requirements, which are summarized in
+# [Kennedy2001](@cite).
+
+# A common setting is the case ``N = 2``. This gives the typical context for
+# Implicit-Explicit (IMEX) splitting methods, where the tendency ``{\mathcal{T}}``
+# is assumed to have the decomposition:
+#
+# ``
+# 	\dot{\boldsymbol{q}} = \mathcal{T}(\boldsymbol{q}) \equiv
+# 	{\mathcal{T}}_{s}(\boldsymbol{q}) + {\mathcal{T}}_{ns}(\boldsymbol{q}),
+# ``
+# where the right-hand side has been split into a "stiff" component ``{\mathcal{T}}_{s}``,
+# to be treated implicitly, and a non-stiff part ``{\mathcal{T}}_{ns}`` to be treated explicitly.
+
+# Referencing the canonical form introduced in [`Time integration`](@ref
+# Time-integration) we have that in this particular forumlation
+# ``\mathcal{T}_{ns}(t, \boldsymbol{q}) \equiv \mathcal{G}(t, \boldsymbol{q})`` and
+# ``\mathcal{T}_{s}(t, \boldsymbol{q}) \equiv \mathcal{F}(t, \boldsymbol{q})``.
+
+# Two different RK methods are applied to ``{\mathcal{T}}_{s}`` and ``{\mathcal{T}}_{ns}``
+# separately, which have been specifically designed and coupled. Examples can be found in
+# [Giraldo2013](@cite). The Butcher Tableau for an `ARK_2` method will have the
+# form
+#
+# ```math
+# \begin{align}
+#     \begin{array}{c|c|c}
+#     \boldsymbol{c} &\boldsymbol{A}_E &\boldsymbol{A}_I\\
+#     \hline
+#     & \boldsymbol{b}_E^T & \boldsymbol{b}_I^T \\
+#     \hline
+#     & \widehat{\boldsymbol{b}}_E^T & \widehat{\boldsymbol{b}}_I^T
+#     \end{array},
+# \end{align}
+# ```
+#
+# with
+#
+# ``
+# 	\boldsymbol{A}_O = \left\lbrace a_{i, j}^O \right\rbrace, \quad
+# 	\boldsymbol{b}_O = \left\lbrace b_{i}^O \right\rbrace, \quad
+# 	\widehat{\boldsymbol{b}}_O = \left\lbrace \widehat{b}_{i}^O \right\rbrace,
+# ``
+#
+# where ``O`` denotes the label (either ``E`` for explicit or ``I`` for implicit).
+
+# For the acoustic wave example used here, we use 4th order polynomials in
+# our discontinuous Galerkin approximation, with 6 elements in each horizontal
+# direction and 4 elements in the vertical direction, on the cubed-sphere.
+# This gives an effective minimanl node-distance (distance between LGL nodes)
+# of roughly 203000 m.
+# As in the [previous tutorial](@ref Single-rate-Explicit-Timestepping),
+# we can determine our ``\Delta t`` by specifying our desired horizontal
+# Courant number ``C`` (the timestep calculation is based on the CFL condition
+# for horizontally-propogating acoustic waves). In this very simple test case,
+# we can use a value of 0.5, which corresponds to a time-step size of
+# around 257 seconds. But for this particular example, even higher values
+# might work.
+
+timeend = FT(3600)
+ode_solver = ClimateMachine.IMEXSolverType(
+    solver_method = ARK2GiraldoKellyConstantinescu,
+)
+C = FT(0.5)
```

Comment:
CFL

Reply:
Thank you, I am going to change all its occurrences